Bixal/move.mil
# Drupal core updates

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Updated drupal/core and other composer packages
- Updated npm packages

## Testing

To verify the changes proposed in this pull request…

1. go to project root, check drupal version. Should be 8.6.14
2. check website that nothing looks or acts different
